packages/cmatrix.rb

```diff
@@ -3,24 +3,15 @@
 class Cmatrix < Package
   description "CMatrix is a program to see the cool scrolling lines from 'The Matrix' movie."
   homepage 'http://www.asty.org/cmatrix/'
-  version '1.2a-1'
```

Comment:
Why are we moving from 1.2a to 1.2? Isn't that going backwards or is this a cmatrix thing?